### PR TITLE
#5152 import public google spreadsheet with editors information 

### PR DESCRIPTION
**Pull request by tayowonibi**
resolves https://github.com/elifesciences/issues/issues/5152
- import public editor profile based on new schema
- use same timestamp for all sheets in the spreadsheet imported

### CODE REVIEW

File: flows/editors-google-spreadsheet-deposit/src/editorscsv2json.py [hunk 1]
```diff
@@ -52,4 +54,5 @@ def main(input=None, output=None, filename=None):
 
 if __name__ == "__main__":
     args = sys.argv[1:]
-    main(filename=first(args) or None)
+    file_name, current_time = get_args(args)
```
Comment (de-code) [inline flows/editors-google-spreadsheet-deposit/src/editorscsv2json.py]: I would maybe recommend changing it more to something like _get with default_ (if `argparse` seems too much). (`get_args` doesn't really give away much about what it's doing - we already have `args`)

And then maybe use it a bit like:

```python
file_name = get_at(args, 0, None)
current_time = get_at(args, 1, now())
```

or

```python
current_time = get_at(args, 1, None) or now()
```

**Comment by de-code** [comment]
I guess we can't ask NiFi to give us the time in UTC?

**Review by de-code — APPROVED** [review]

File: flows/editors-google-spreadsheet-deposit/src/editorscsv2json.py (hunk 1 above)
Comment (tayowonibi) [reply flows/editors-google-spreadsheet-deposit/src/editorscsv2json.py]: done

**Comment by tayowonibi** [comment]
> I guess we can't ask NiFi to give us the time in UTC?

yes

**Pull request merged**